waku/node/peer_manager/peer_manager.nim

```diff
@@ -101,19 +101,15 @@ proc calculateBackoff(initialBackoffInSec: int,
 # Helper functions #
 ####################
 
-proc insertOrReplace(ps: PeerStorage,
-                     peerId: PeerID,
-                     remotePeerInfo: RemotePeerInfo,
-                     connectedness: Connectedness,
-                     disconnectTime: int64 = 0) =
-  # Insert peer entry into persistent storage, or replace existing entry with updated info
-  let res = ps.put(peerId, remotePeerInfo, connectedness, disconnectTime)
-  if res.isErr:
-    warn "failed to store peers", err = res.error
+proc insertOrReplace(ps: PeerStorage, remotePeerInfo: RemotePeerInfo) =
+  ## Insert peer entry into persistent storage, or replace existing entry with updated info
+  ps.put(remotePeerInfo).isOkOr:
+    warn "failed to store peers", err = error
     waku_peers_errors.inc(labelValues = ["storage_failure"])
+    return
 
 proc addPeer*(pm: PeerManager, remotePeerInfo: RemotePeerInfo, origin = UnknownOrigin) =
-  # Adds peer to manager for the specified protocol
+  ## Adds peer to manager for the specified protocol
```

Comment:
Any specific reason? In other parts of the PR you changed from 2 #s to one, here it's the opposite haha

Reply:
Two `##` after the proc declaration is picked up by the tooling as the proc "explanation". One `#` are for comments.